src/MongoDB.Driver.Core/Core/Operations/BulkWriteConcernError.cs

```diff
@@ -56,11 +55,25 @@ public BulkWriteConcernError(int code, string message, BsonDocument details)
         /// <param name="message">The message.</param>
         /// <param name="details">The details.</param>
         public BulkWriteConcernError(int code, string codeName, string message, BsonDocument details)
+            : this(code, codeName, message, details, new string[0])
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BulkWriteConcernError" /> class.
+        /// </summary>
+        /// <param name="code">The code.</param>
+        /// <param name="codeName">The name of the code.</param>
+        /// <param name="message">The message.</param>
+        /// <param name="details">The details.</param>
+        /// <param name="errorLabels">The error labels.</param>
+        public BulkWriteConcernError(int code, string codeName, string message, BsonDocument details, IEnumerable<string> errorLabels)
```

Comment:
Note: this is a new public ctor of public class, and `internal` is sufficient here, so I'm not sure which level suits better here.